WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/MainActivity.kt

```diff
@@ -18,15 +18,23 @@ package com.example.android.wearable.speaker
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import androidx.annotation.OptIn
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.exoplayer.ExoPlayer
 
 /**
  * A simple activity that just serves as an entry point to [SpeakerApp].
  */
 class MainActivity : ComponentActivity() {
+    @OptIn(UnstableApi::class)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        val player = ExoPlayer.Builder(this)
```

Comment:
This is nice. I suspect we want this in a view model scope?

But I'm not sure.

Reply:
it's now in the view model